components/list/list-item-generic-layout.js

```diff
@@ -390,6 +409,13 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 		this._cellNum = parseInt(slot.getAttribute('data-cell-num'));
 		this._cellFocusedItem = this._getFocusedItemPosition(event.target);
 	}
+
+	_unsetFocusInfo() {
+		setTimeout(() => {
+			this._cellNum = null;
+			this._cellFocusedItem = null;
+		}, 100);
```

Comment:
It needs a timeout to account transition animations, which look super weird without this.